### ISSUE TITLE
Please update versions

### ISSUE THREAD

**Issue by frankbenoit** [diffplug/gradle_and_eclipse_rcp#15]
Hello Ned,

I think I could really learn a lot from this example.
Unfortunately it uses really old version of everything.
Gradle wrapper, Goomph, BND, Spotless, PDE

When i update those, the build blocks in the task 'target.p2'.
Can you please do the update? This would increase the value of the example a lot :-)

Frank

**Comment by nedtwigg** [comment]
Here at DiffPlug, we have transitioned off of p2/RCP completely, and we now use the [`eclipse.mavencentral` plugin](https://diffplug.github.io/goomph/javadoc/3.18.0/com/diffplug/gradle/eclipse/MavenCentralPlugin.html) to grab the core eclipse libraries from maven central.  We'll keep merging PRs and offering guidance for people who are using the RCP and P2 plugins, but we aren't going to lead any further investment in this direction.

If I were to do this, I would go step-by-step - update one thing, do a clean build, make sure it works.  Update the next, do a clean build, make sure it works.  When something breaks, I would manually bisect from the known-working version to the known-broken version until I found the last-working version of that particular dependency.

If someone does that work and gets stuck "e.g. goomph 3.12 works but then 3.13 hangs after these other update commits" then I'd be happy to help diagnose and fix the root problem.

**Comment by watho** [comment]
Just want to let you know that the example does not build anymore, cause nodeclipse.org is no valid repository anymore. Instead its a spamsite now.

This example is still referenced from https://github.com/diffplug/goomph for building RCP-applications.

**Comment by nedtwigg** [comment]
Now that #16 has been merged, this example is working with all latest versions as of April 2020.  For me, on Windows 10, it is not generating the launcher correctly, but everything else works without any deprecation warnings.  All I did was bump the eclipse version from `4.5.2` all the way to `4.15.0`, so it may easily be that the product file needs to be updated.  Happy to take PRs to improve this, but keeping this up-to-date is super-uper-duper low priority for me.
